tests/test_grids.py

```diff
@@ -545,8 +545,9 @@ class MyParticle(ptype[mode]):
 
 @pytest.mark.parametrize('mode', ['scipy', 'jit'])
 @pytest.mark.parametrize('vert_discretisation', ['zlevel', 'slevel', 'slevel2'])
-def test_popgrid(mode, vert_discretisation):
-    mesh = path.join(path.dirname(__file__), 'test_data/') + 'POPtestdata_.nc'
+@pytest.mark.parametrize('deferred_load', [True, False])
+def test_popgrid(mode, vert_discretisation, deferred_load):
+    mesh = path.join(path.dirname(__file__), 'test_data/') + 'POPtestdata_time.nc'
```

Comment:
Should the `+` here be a `path.join()` too? This may not work on Windows, where directory structure is with `\` instead of `/` (which is why we use `path.join()` instead)